packages/forms-web-app/src/public/resources/question12/content.html

```diff
@@ -0,0 +1,10 @@
+<p class="govuk-body">We need a:</p>
```

Comment:
naming: this might not always be question 12 - maybe use the fieldname instead?

Reply:
I think this might just be in there as example for now, but fieldName is a good shout. We've got the actual user story here https://pins-ds.atlassian.net/browse/AAPD-194 so we can tweak naming conventions in there.